### PR TITLE
Homework25: Alina Shautsova

### PR DESCRIPTION
**Pull request by AlinaShautsova**

### CODE REVIEW

File: homework25/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    """The browser fixture to open the login page."""
+
+    browser = open_browser
+    wait = WebDriverWait(browser, 10)
+
+    login_page = LoginPage(browser)
+    login_page.login(EMAIL, PASSWORD)
+
+    wait.until(EC.presence_of_element_located(
+        (By.XPATH, '//*[contains(text(), "Contact List")]')))
```
Comment (Stml89) [inline homework25/tests/conftest.py]: move to locators

File: homework25/pages/add_user_page.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+from homework25.test_data.add_user_test_data import URL
+from homework24.locators.add_user_page import AddUserPageLocators
+
+
+class AddUserPage(BasePage):
+    """Class Add user."""
+    def __init__(self, driver):
+        """Initialisation."""
+        super().__init__(driver)
+        self.url = URL
```
Comment (Stml89) [inline homework25/pages/add_user_page.py]: do you really need URL?

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework25/tests/conftest.py (hunk 1 above)
Comment (AlinaShautsova) [reply homework25/tests/conftest.py]: Moved

File: homework25/pages/add_user_page.py (hunk 2 above)
Comment (AlinaShautsova) [reply homework25/pages/add_user_page.py]: Removed

**Review by Stml89 — APPROVED** [review]

**Pull request merged**